packages/compass-preferences-model/src/preferences.ts

```diff
@@ -119,10 +104,11 @@ type PostProcessFunction<T> = (
 ) => T;
 
 type PreferenceDefinition<K extends keyof AllPreferences> = {
-  /** The type of the preference value, in Ampersand naming */
-  type: AmpersandType<AllPreferences[K]>;
+  /** The type of the preference value */
+  type: PreferenceValue<AllPreferences[K]>;
```

Comment:
Did Ampersand do validation of the type? I think it did, that's missing now, right?